src/firefly/js/visualize/ui/FileUploadViewPanel.jsx

```diff
@@ -72,35 +55,43 @@ const TABLES_ONLY_SUPPORTED_TYPES=[
     FileAnalysisType.Table,
 ];
 
-
-const isTablesOnly= () => getAppOptions()?.uploadPanelLimit==='tablesOnly';
-
-
 const uploadOptions = 'uploadOptions';
 
-let currentAnalysisResult, currentReport, currentSummaryModel, currentDetailsModel;
 const FILE_UPLOAD_KEY= 'file-upload-key-';
 let keyCnt=0;
 
-export function FileUploadViewPanel({setSubmitText}) {
+
+export function FileUploadViewPanel({setSubmitText, showCompleteButton}) {
```

Comment:
I don't understand why you want `FileUploadViewPanel` to manage the complete button. I don't think you should have it.